.circleci/config.yml

```diff
@@ -611,6 +611,40 @@ jobs:
           command: npx depcheck
           working_directory: integration-tests
 
+  atst-tests:
+    docker:
+      - image: ethereumoptimism/ci-builder:latest
+    resource_class: large
+    steps:
+      - checkout
+      - attach_workspace: { at: '.' }
+      - check-changed:
+          patterns: atst,contracts-periphery
+      - run:
+          name: anvil
+          background: true
+          command: anvil --fork-url $ANVIL_L2_FORK_URL --fork-block-number 92093723
+      - run:
+          name: build
+          command: yarn build
+          working_directory: packages/atst
+      - run:
+          name: typecheck
+          command: yarn typecheck
+          working_directory: packages/atst
+      - run:
+          name: lint
+          command: yarn lint:check
+          working_directory: packages/atst
+      - run:
+          name: make sure anvil is up
+          command: npx wait-on tcp:8545 && cast block-number --rpc-url http://localhost:8545
```

Comment:
@zhwrd this time I added this `npx wait-on tcp:8545` to make sure there is no flaky race condition and I also added a `--fork-block-number` to make the ethereum state deterministic and prevent any flake happening from the test conditions changing out from under us

Reply:
I think last time I was here we found out that the issue was CMD vs entrypoint so I'm happy to try the docker way again if preferred